spec/features/step_by_step_navigation_spec.rb

```diff
@@ -9,6 +9,7 @@
     before do
       given_I_am_a_GDS_editor
       setup_publishing_api
+      stub_request(:put, %r{/publish-intent/}) # stub any PUT calls made by the scheduled publish worker
```

Comment:
Change comment to say `StepByStepScheduledPublishWorker` so it's copy & pasteable and searchable